dushimeemma/fs-simba-cal
# [WIP] Implement booking endpoint

Description
Type of change
[ ] Bug fix (Non breaking change that fixes an existing issue)

[ ] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested?
[ ] Unit tests

[ ] Component Tests

[ ] Integration Tests

How Should This Be Manually tested:

- Clone this repo `git clone https://github.com/dushimeemma/fs-simba-cal/`.

- Checkout the branch `git checkout branch_name`

- Run `yarn install`

- Run `yarn dev`

Checklist:
[ ] My code follows all the styles and conventions outlined for this project

[ ] I have linted my code before submission

[ ] I have made my corresponding changes to the documentation

[ ] My changes generates no new warning

[ ] I have added tests that prove my feature is working or my fix is effective

[ ] New and existing tests pass locally with my changes

JIRA
issue-id
